code/modules/organs/subtypes/standard.dm

```diff
@@ -268,6 +268,11 @@
 
 	var/eye_icon_location = 'icons/mob/human_face.dmi'
 
+/obj/item/organ/external/head/Initialize()
+	..()
```

Comment:
Generally ``Initialize()`` needs to return a thing and so usually at the bottom is ``return ..()``, similar to ``Destroy()``.

Reply:
Righto.